lib/workers/bin/run_single_worker.rb

```diff
@@ -81,7 +81,7 @@ def all_role_names
 # Skip heartbeating with single worker
 ENV["DISABLE_MIQ_WORKER_HEARTBEAT"] ||= options[:heartbeat] ? nil : '1'
 
-options[:ems_id] ||= ENV["EMS_ID"]
+options[:ems_id] ||= Array.wrap(ENV["EMS_ID"])
```

Comment:
Note, this doesn't support setting multiple EMS_ID values from ENV as nothing splits on "," or anything else.
Since, ENV values can only be nil or a String, I wonder if we should assume a string and if support for comma separated values is needed:

```ruby
options[:ems_id] ||= ENV['EMS_ID'].try(:split, ',')
```

For example:
```ruby
joerafaniello@Joes-MBP-2 manageiq % EMS_ID="1" ruby -r './config/environment'  -e "puts ENV['EMS_ID'].try(:split, ',').inspect"
** ManageIQ master, codename: Lasker
["1"]
joerafaniello@Joes-MBP-2 manageiq % EMS_ID="1,2" ruby -r './config/environment'  -e "puts ENV['EMS_ID'].try(:split, ',').inspect"
** ManageIQ master, codename: Lasker
["1", "2"]
joerafaniello@Joes-MBP-2 manageiq %  ruby -r './config/environment'  -e "puts ENV['EMS_ID'].try(:split, ',').inspect"
** ManageIQ master, codename: Lasker
nil
```

Reply:
:+1:

Reply:
Also, note Array.wrap of a nil will be `[]` and the code below checks `if options[:ems_id]` which will be true for `[]`

```
joerafaniello@Joes-MBP-2 manageiq %  ruby -r './config/environment'  -e "puts Array.wrap(ENV['EMS_ID']).inspect"
** ManageIQ master, codename: Lasker
[]
```

Reply:
Meanwhile... missing env keys will be nil:

```
irb(main):001:0> puts ENV['blah'].inspect
nil
```

so, this would change this line 84 to from `nil` to `[]` in the event EMS_ID is not set in ENV.

Reply:
Done, and I'll work on a follow-up to change this to not work on arrays since we shouldn't be passing those in anymore